airflow-core/docs/core-concepts/auth-manager/index.rst

```diff
@@ -178,6 +180,8 @@ The following methods aren't required to override to have a functional Airflow a
 
 * ``batch_is_authorized_dag``: Batch version of ``is_authorized_dag``. If not overridden, it will call ``is_authorized_dag`` for every single item.
 * ``get_authorized_dag_ids``: Return the list of DAG IDs the user has access to.  If not overridden, it will call ``is_authorized_dag`` for every single DAG available in the environment.
+  * Note: To filter the results of ``get_authorized_dag_ids``, it is recommended that you define the filtering logic in your ``filter_authorized_dag_ids`` method. For example, this may be usefule if you rely on per-DAG access controls derived from one or more fields on a given DAG (e.g. DAG tags).
```

Comment:
Nice catch from the spellcheck

```suggestion
  * Note: To filter the results of ``get_authorized_dag_ids``, it is recommended that you define the filtering logic in your ``filter_authorized_dag_ids`` method. For example, this may be useful if you rely on per-DAG access controls derived from one or more fields on a given DAG (e.g. DAG tags).
```